src/libServer/IsolatedServer.cpp

```diff
@@ -1181,9 +1181,13 @@ TxBlock IsolatedServer::GenerateTxBlock() {
     m_txnBlockNumMap[m_blocknum].clear();
   }
 
+  auto const prevTxBlock = m_sharedMediator.m_txBlockChain.GetLastBlock();
+
+  auto const prevHash = m_blocknum == 0 ? BlockHash(): prevTxBlock.GetBlockHash();
+
   TxBlockHeader txblockheader(0, m_currEpochGas, 0, m_blocknum,
                               TxBlockHashSet(), numtxns, m_key.first,
-                              TXBLOCK_VERSION);
+                              1, TXBLOCK_VERSION, CommitteeHash(), prevHash);
```

Comment:
this was actually incedentally incorrectly setting the version as the ds block number here